docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -536,6 +536,29 @@ Additionally, some amount of back-pressure is both _normal_ and _expected_ for p
 
 |===
 
+Each flow stat includes rates for one or more recent windows of time:
+
+// Templates for short-hand notes in the table below
+:flow-stable: pass:quotes[*Stable*]
+:flow-preview: pass:quotes[_Technology Preview_]
+
+[%autowidth.stretch]
+|===
+| Flow Window       | Availability   | Definition
+
+| `current`         | {flow-stable}  | the most recent ~10s
+| `lifetime`        | {flow-stable}  | the lifetime of the relevant pipeline or process
```

Comment:
DOCS: Would it be helpful to indicate when the stable windows were introduced?

```suggestion
| `current`         | {flow-stable} since 8.5 | the most recent ~10s
| `lifetime`        | {flow-stable} since 8.5 | the lifetime of the relevant pipeline or process
```